content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -114,6 +114,35 @@ stages:
 The commands in `dvc metrics` and `dvc plots` help you display and compare
 metrics and plots.
 
+## Plots definitions
```

Comment:
Thanks for adding this section @pared. "Pipelines files" is no longer a good descriptor for this page if we include it, but I'm not sure there is a better place for it now, and I would say let's not let it block this PR for now.